src/pages/intervention-general/intervention-general.ts

```diff
@@ -219,4 +235,23 @@ export class InterventionGeneralPage implements OnDestroy {
                     });
         }
     }
+
+    private async canUserAccessInspection(){
+        if(this.plan.status != this.inspectionDetailProvider.InspectionStatusEnum.Started){
```

Comment:
Je crois bien que ce IF est problématique.

Si l'inspection n'est pas commencée, ça écrit 
* Pending inspection
* This inspection is in progress by another user.